### PR TITLE
Tweak keeper for alerts

### PR DESCRIPTION
**Pull request by ebatsell**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -27,12 +27,40 @@ services:
       - METRICS_INTERVAL=${METRICS_INTERVAL}
       - RUN_STAKE_UPLOAD=${RUN_STAKE_UPLOAD}
       - RUN_GOSSIP_UPLOAD=${RUN_GOSSIP_UPLOAD}
-      - RUN_EMIT_METRICS=${RUN_EMIT_METRICS}
+      - RUN_EMIT_METRICS=false
```
Comment (coachchucksol) [inline docker-compose.yaml:30]: You can specify this in the .env

File: keepers/validator-keeper/src/entries/crank_steward.rs [hunk 2]
```diff
@@ -122,31 +122,33 @@ pub fn _get_update_stake_pool_ixs(
...
-
-            let vote_account = VoteState::deserialize(&raw_vote_account.clone().unwrap().data)
-                .expect("Could not deserialize vote account");
-
-            let latest_epoch = vote_account.epoch_credits.iter().last().unwrap().0;
-
-            match stake_account {
-                StakeStateV2::Stake(_meta, stake, _stake_flags) => {
-                    if stake.delegation.deactivation_epoch != std::u64::MAX {
+        let should_deactivate = match (raw_vote_account, raw_stake_account) {
```
Comment (coachchucksol) [inline keepers/validator-keeper/src/entries/crank_steward.rs:125]: This is much better!

File: sdk/src/utils/transactions.rs [hunk 3]
```diff
@@ -650,7 +669,7 @@ pub async fn submit_transactions(
         .map(|t| t.as_slice())
         .collect::<Vec<_>>();
 
-    match parallel_execute_transactions(client, &tx_slice, keypair, 100, 20).await {
+    match parallel_execute_transactions(client, &tx_slice, keypair, 100, 30).await {
```
Comment (coachchucksol) [inline sdk/src/utils/transactions.rs:672]: The 100 and 30 _should_ be args

Comment (ebatsell) [reply sdk/src/utils/transactions.rs:672]: adding

**Pull request merged**